services/vision/mlvision/classifier.go

```diff
@@ -22,8 +22,14 @@ func attemptToBuildClassifier(mlm mlmodel.Service) (classification.Classifier, e
 
 	// Set up input type, height, width, and labels
 	var inHeight, inWidth uint
+	if len(md.Inputs) < 1 {
+		return nil, errors.New("could not get information about first input tensor")
```

Comment:
I think this should say something like "the length of input tensors are 0" or "no input tensors received"